ui/app/AppLayouts/Timeline/TimelineLayout.qml

```diff
@@ -33,7 +38,7 @@ ScrollView {
         if(parentPopup){
             popup.parentPopup = parentPopup;
         }
-        popup.openPopup(profileModel.profile.pubKey !== fromAuthorParam, userNameParam, fromAuthorParam, identiconParam, textParam, nicknameParam);
+        popup.openPopup(root.store.profileModelInst.profile.pubKey !== fromAuthorParam, userNameParam, fromAuthorParam, identiconParam, textParam, nicknameParam);
```

Comment:
Property reads are still done view property lookups right? Or do we introduce getter proxy APIs for that on stores as well? @alexandraB99